src/base/bittorrent/session.cpp

```diff
@@ -4009,6 +3990,12 @@ void Session::handleAlert(libt::alert *a)
         case libt::add_torrent_alert::alert_type:
             handleAddTorrentAlert(static_cast<libt::add_torrent_alert*>(a));
             break;
+        case libt::torrent_added_alert::alert_type:
+            if (m_startupInProgress)
```

Comment:
Please don't break the current style. Just add `handleTorrentAddedAlert()` method.